packages/volto/cypress/tests/core/blocks/blocks-grid.js

```diff
@@ -22,7 +22,7 @@ context('Blocks Acceptance Tests', () => {
         contentType: 'Image',
         contentId: 'my-image',
         contentTitle: 'My Image',
-        path: '/document',
```

Comment:
@sneridagh I don't understand why the path is empty

Reply:
Me neither... maybe a trial? what I don't get is that it still works...